pydantic/_internal/_generate_schema.py

```diff
@@ -821,10 +821,18 @@ def _generate_schema_from_property(self, obj: Any, source: Any) -> core_schema.C
         ):
             schema = existing_schema
         elif (validators := getattr(obj, '__get_validators__', None)) is not None:
-            warn(
-                '`__get_validators__` is deprecated and will be removed, use `__get_pydantic_core_schema__` instead.',
-                PydanticDeprecatedSince20,
-            )
+            from pydantic.v1 import BaseModel as BaseModelV1
+
+            if issubclass(obj, BaseModelV1):
```

Comment:
```suggestion
            if inspect.isclass(obj) and issubclass(obj, BaseModelV1):
```
Because `issubclass` raises a `TypeError` if `obj` is not a class. But I'm wondering if it's possible for `obj` to have `__get_validators__` and not being a class in this context, so probably not necessary to add the `isclass` check.